CHANGELOG.md

```diff
@@ -25,9 +156,9 @@ be found.
 
 ### Security
 
-- Fix linux32 emulation to fail during docker build [#20672](https://github.com/docker/docker/pull/20672)  
```

Comment:
Oh, think the whitespace here and below got lost; these were deliberate to format the output